### DISCUSSION TITLE
Deep-link URL for entity details

### DISCUSSION THREAD

**Discussion by svh1985** [home-assistant/frontend#10403] [Dashboard]
I would like to request the ability to deep-link to an entity by URL.

Example: `https://home.assistant.com/view/entity_id=camera.garage` or via companion app: `homeassistant://view/entity_id=camera.garage `which opens the entity detail page.

Case:
Send camera notification to app or browser. Tap on the notification to open Home Assistant which navigates to the live camera view.

This is of course also use full for other notifications.

ref: At this point, it is only possible to navigate to a Lovelace dashboard with the companion app: https://companion.home-assistant.io/docs/integrations/url-handler/

**Comment by jaymemaurice** [comment]
This would be extremely useful for notifications.

**Comment by drewsteinacher** [comment]
I was able to successfully do this via this feature that appears to have been quietly released in 2025.6:
https://github.com/home-assistant/frontend/pull/25733

Relevant Community thread comment how I found it:
https://community.home-assistant.io/t/direct-url-to-entity-details-user-interface/463280/23

TL;DR: You can just add a `more-info-entity-id=` query param to any dashboard URL and it will pull up the dialog.

In my case, I used an Android automation app to open this URL on a widget button press, which did the job nicely:
```
homeassistant://navigate/lovelace/0?more-info-entity-id=timer.whatever
```
